docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/RandomTrainerSample.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Data;
+using Microsoft.ML.Trainers;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class RandomTrainerSample
```

Comment:
>RandomTrainerSample [](start = 17, length = 19)

Zeeshan A, Shahab and me have PRs where we create BinaryClassification folder.
Any chance you can move this two to that folder?